scikeras/utils/transformers.py

```diff
@@ -55,6 +76,11 @@ def fit(self, X: np.ndarray) -> "ClassifierLabelEncoder":
                 "\n\nTo implement support, subclass KerasClassifier and override"
                 " `target_transformer` with a transformer that supports this"
                 " label type."
+                "\n\nFor information on sklearn target types, see:"
+                " * https://scikit-learn.org/stable/modules/generated/sklearn.utils.multiclass.type_of_target.html"
+                " * https://scikit-learn.org/stable/modules/multiclass.html"
+                "\n\nFor information on the SciKeras data transformation interface, see:"
+                " * https://scikeras.readthedocs.io/en/latest/advanced.html#data-transforms"
```

Comment:
Maybe it's a better idea to have SciKeras docs page to collect these links and describe how to fix the problem. I've added this to #68.

Reply:
I agree we should have a doc page that walks users through the data transform system were implementing here, and how to fix these NotImplementedErrors. But to be clear, are you suggesting _also_ removing the links from here?

Reply:
> But to be clear, are you suggesting also removing the links from here?

It's up to you; I'm indifferent as it's tracked in #68.

Reply:
I'll leave it. I think it's nice to have the links in the error.